File: frontend/src/types/owner.ts
export type Owner = {
  idOwner?: string;           
  name: string;
  address: string;
  photo: string;
  birthday: string;       
};

export type OwnerList = {
  idOwner: string;
  name: string;
};

export type CreateOwnerRequest = {
  name: string;
  address: string;
  photo: string;
  birthday: string;       
};

File: backend/Properties.Domain/Entities/Owner.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Properties.Domain.Entities
{
    public class Owner
    {
        public Guid IdOwner { get; set; } = default!;
        public string Name { get; set; } = default!;
        public string Address { get; set; } = default!;
        public string Photo {  get; set; } = default!;
        public DateTime Birthday { get; set; }
    }
}
